src/main/resources/reference.conf

```diff
@@ -167,6 +167,12 @@ sparkz {
     # Limit for number of modifiers to request and process at once
     maxRequestedPerPeer = 1024
 
+    # Enables or disables slow mode - ignoring requesting or broadcasting new transaction when node is overloaded
+    slowModeFeatureFlag = false
+
+    # Threshold of average time it takes node to apply a new modifier, after which we consider it overloaded and start throttling
+    slowModeThresholdMs = 2000
```

Comment:
This value is unique for transactions and blocks. Consider that a reasonable value for a block processing time is quite higher than the time needed for a transaction processing, it means that the estimation of when a node is busy is biased from the amount of transaction received.
 `averageProcessingTimeMs = (averageProcessingTimeMs * 0.9).toLong + (elapsedMs * 0.1).toLong`

Reply:
Ok but in this case why not ignore the block processing time? I think the point here is that we are mixing 2 different things no?

Reply:
I want to also keep blocks, because even though the blocks will be processed slower, both are processed by a single consecutive actor - NodeViewHolder - so if it takes 1 second to process a transaction, but 3 seconds to process a block, block should affect the average, because while block is being processed, all transactions are waiting in queue and will potentially timeout.

What I am trying to say is that the queue is important. Lets consider this example - it takes 1 second to process a block, and 0.1 seconds to process a transaction:
```
relative time - action
0.0  - 2tx recieved and sent to NVH
0.5  - 1 block recieved and sent to NVH
1.0  - 2tx recieved and sent to NVH

we will get next measurements:
0.1  - tx1 processed - measured 0.1s
0.2  - tx2 processed - measured 0.2s
1.5  - block processed - measured 1.0s
1.6  - tx3 processed - measured 1.1s
1.7  - tx4 processed - measured 1.2s
```
So the general idea is that we are mostly interested in the total time a modifiers requires to be applied (time in queue + processing time), and time in queue is the dominant here